app/vmauth/main.go

```diff
@@ -56,6 +56,8 @@ var (
 		"See https://docs.victoriametrics.com/vmauth.html#backend-tls-setup")
 	backendTLSCAFile = flag.String("backend.TLSCAFile", "", "Optional path to TLS root CA file, which is used for TLS verification when connecting to backends over HTTPS. "+
 		"See https://docs.victoriametrics.com/vmauth.html#backend-tls-setup")
+	httpAuthHeaders = flagutil.NewArrayString("httpAuthHeader", "alternative to Authorization header names that can be an auth token source")
```

Comment:
I'd like to rethink current naming.

1) if it adds additional headers to the list of pre-defined values: `extraAuthHeaderName`. 
2) if it could replace default values: `authHeaderName` ( set to `Authorization` by default).